sympy/simplify/radsimp.py

```diff
@@ -169,6 +169,10 @@ def collect(expr, syms, func=None, evaluate=None, exact=False, distribute_order_
 
     collect_const, collect_sqrt, rcollect
     """
+    _eval_collect = getattr(expr, '_eval_collect', None)
+    if _eval_collect is not None:
+        return _eval_collect(syms, func, evaluate,
+                             exact, distribute_order_term)
```

Comment:
Where did this change came from?
I don't think that we are using `_eval_collect` anyway
https://github.com/search?q=repo%3Asympy%2Fsympy%20_eval_collect&type=code

Reply:
Is this resolved for now?